src/content/guides/ecommerce-website.mdx

```diff
@@ -20,21 +22,86 @@ Centrapay will provide you with the following resources to allow you to start cr
 
 > We strongly recommend Centrapay APIs are invoked from your backend where your API key is securely stored.
 
-### Access the Centrapay SDK
-
-The Centrapay SDK enables acceptance of Centrapay payments on your website.
-It handles displaying the Centrapay button and launching the Centrapay checkout.
-
-Production: `https://sdk.centrapay.com/ecommerce/centrapay.js?merchantId={merchantId}`
+Centrapay will configure your business to accept test payments before you can accept live payments.
+Accepting live payments requires you to meet our certification requirements.
 
-### Centrapay Checkout Sample
+## Centrapay Checkout Sample
 
 A [sample Centrapay e-commerce application](https://github.com/centrapay/centrapay-checkout-sample) is available.
-It uses the Centrapay SDK on the frontend and Node.js on the backend.
+It includes demos of both the redirect method and the popup method.
 Please refer to the README.md file for configuration instructions.
 
+## Redirect Method
+
+The Redirect Method is the standard method used by most merchants.
```

Comment:
Do we mention that this is the safer method?

Reply:
@alexBinns safer how?